CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ## Unreleased
 
+### Changed
+
+- Timestamp argument accepts input in string type( format `YYYY-MM-DD HH:MM:SS` ), int type(epoch time) as well as a datetime instance for following:
+    - `sdk.auditlogs.get_page`
+    - `sdk.auditlogs.get_all`
```

Comment:
It is typical for us to end methods with `()`, so like `sdk.securitydata.get_all_plan_security_events()` etc.

Reply:
Changed.